### DISCUSSION TITLE
Visual Studio Code - marketplace 网站图标显示方块

### DISCUSSION THREAD

**Discussion by GeBron** [F9y4ng/GreasyFork-Scripts#382] [问答专区(Question / Answer)] [answered]
URL： https://marketplace.visualstudio.com/items?itemName=redhat.java

![微信截图_20241021153046](https://github.com/user-attachments/assets/a3ecf351-e3c9-4626-8d13-7592f1f64329)

**Comment by F9y4ng** [comment, marked as answer]
图标显示与否与脚本无关，和网络有关系。

因为图标字体没有成功下载，所以显示为方块，建议遇到这样的情况，全局翻墙解决。（或marketplace.visualstudio.com局部翻）

如果logo地址被解析到 `vsassetscdn.azure.cn` 域上，基本就宣布凉凉。如图：

![screenshot_2024-10-21_19-36-53](https://github.com/user-attachments/assets/e36270c4-1a6a-4c06-ad34-25a52de52153)

正常解析，如图：

![screenshot_2024-10-21_19-42-25](https://github.com/user-attachments/assets/67856183-96ed-44ac-888f-196c110606f0)
